requirements/base.txt

```diff
@@ -4,4 +4,12 @@
 #
 #    pip-compile --output-file requirements/base.txt requirements/base.in
 #
-Django==1.10.5
+amqp==2.1.4               # via kombu
+billiard==3.5.0.2         # via celery
+celery==4.0.2
+django-model-utils==2.6.1
+Django==1.10.5            # via django-model-utils, jsonfield
```

Comment:
See base.in as well as tox.ini and .travis.yml.  Tests are run against the latest versions of django 1.8, 1.9, and 1.10.